src/pages/ThorChainLP/AvailablePools.tsx

```diff
@@ -106,23 +98,27 @@ const PoolButton = ({ pool }: PoolButtonProps) => {
         </Box>
         <RawText>{pool.name}</RawText>
         <Tag size='sm'>
-          <Amount.Percent value={pool.poolAPY} />
+          <Amount.Percent value={pool.annualPercentageRate} />
         </Tag>
         <Skeleton isLoaded={!isTradingActiveLoading}>
-          {isTradingActive === false ? (
-            <Tag colorScheme='yellow'>
-              <Text translation='common.halted' />
-            </Tag>
-          ) : (
+          {isTradingActive === true && pool.status === 'available' && (
             <Tag colorScheme='green'>
               <Text translation='common.available' />
             </Tag>
           )}
+          {isTradingActive === true && pool.status === 'staged' && (
```

Comment:
show staged (yellow) now, and update halted to red

Reply:
sure! will let @reallybeard and product provide official verbiage, but adding placeholder

Reply:
![image](https://github.com/shapeshift/web/assets/35275952/509e41a3-7a04-498e-8b42-518a1c26f499)

Reply:
https://github.com/shapeshift/web/pull/6278/commits/25f4bc934dea92f78d6d19bccc81565c34061e3e